.scss-lint.yml

```diff
@@ -188,7 +188,7 @@ linters:
 
   StringQuotes:
     enabled: true
-    style: single_quotes # or double_quotes
+    style: double_quotes
```

Comment:
If pronto complains a lot it isn't a solution to just change the config. If you think some of the settings don't make sense and we should change this, we can discuss this, but not as a drive-by of this PR.

Especially this one affects all existing files which previously were created with the old rule.